src/components/percentage/percentage.css

```diff
@@ -0,0 +1,51 @@
+@import '../../styles/colors-v2.css';
+@import "../../styles/spacing.css";
+@import "../../styles/typography-v2.css";
+
+.icons-container {
+  composes: icons from "../form-control-icons/form-control-icons.css";
+  pointer-events: none;
+}
+
+.input {
```

Comment:
Should this stylesheet compose from the MDS input?

Reply:
Probably!